docs/contribute/contributing.md

```diff
@@ -67,14 +64,14 @@ process. We use [the standard DCO text of the Linux Foundation](https://develope
 
 * Fork and clone the repository to your local machine.
 
-```shell
-git clone git@github.com:ironcore-dev/ironcore-dev.github.io.git
-cd docs
-```
+    ```shell
```

Comment:
Do we really need an indent here?

Reply:
But I would rather not indent the code blocks. It makes it later super hard to change things.

Reply:
Yeah, you are right. Removed the identation